src/commands/chatInput/bean.ts

```diff
@@ -0,0 +1,28 @@
+import { ApplicationCommandOptionType } from "discord.js";
+import type { ChatInputCommand } from ".";
+import Emojis from "../../constants/emojis";
+
+const command: ChatInputCommand = {
+  description: "Bean a user (Staff Only)",
```

Comment:
`(Staff Only)` is only for the strip command as that's the only command that will appear publicly available since stripped staff needs to unstrip as well. so tldr remove this